c/src/nnstreamer-capi-util.c

```diff
@@ -198,6 +225,7 @@ ml_tensors_info_validate (const ml_tensors_info_h info, bool *valid)
   if (!tensors_info || tensors_info->num_tensors < 1)
```

Comment:
best to split this check into two parts and move the latter inside the lock.

Reply:
split done.